client/navigation/action-creators.ts

```diff
@@ -8,15 +8,15 @@ export function goToIndex(transitionFn = push): ThunkAction {
   return (_, getState) => {
     const {
       lobby,
-      chat: { idToInfo, joinedChannels },
+      chat: { idToBasicInfo, joinedChannels },
       users: { byId },
       whispers: { sessions },
     } = getState()
     if (lobby.inLobby && IS_ELECTRON) {
       transitionFn(urlPath`/lobbies/${lobby.info.name}`)
     } else if (joinedChannels.size) {
       const [first] = joinedChannels.values()
-      transitionFn(urlPath`/chat/${first}/${idToInfo.get(first)?.name ?? ''}`)
+      transitionFn(urlPath`/chat/${first}/${idToBasicInfo.get(first)?.name ?? ''}`)
```

Comment:
Not a change in this PR, but generally I would replace undefined values with *something*, otherwise you risk problems in adding further parts to the URL in the future. For leagues we use `_` I think. You could pick something else here if you don't want to clash with actual channels/urls (I don't know that we allow 1 character ones anyway?)

Reply:
> I don't know that we allow 1 character ones anyway?

We do, and `_` is a valid channel name :p

I still think `_` is okay to use here, since the name is only decorative in the URL and will be corrected to its real name once the channel page opens.